reana-ui/src/pages/workflowList/components/Welcome.js

```diff
@@ -79,6 +82,9 @@ export function WelcomeNoTokenMsg() {
         Your access token request has been forwarded to REANA administrators.
       </p>
       <Button content="Token requested" disabled />
+      <small className={styles.created}>
+        <em>{moment.utc(tokenCreatedAt).format("Do MMM YYYY HH:mm [UTC]")}</em>
```

Comment:
[No need to change now] It is a bit confusing that for the "Token requested" we use `tokenCreatedAt`, right?

Reply:
I agree, we're taking advantage of the `created` DB column as it matches the request time. Basically we create a new token object when it is requested for the first time. So I propagated the naming but it looks weird for the ui. I can rename it when unpacking the values from the server-side. What about...
- `tokenRequestedAt`
- `tokenRequestTime`
- ...
?

Reply:
`tokenRequestedAt` +1